docs/router/query-batching.mdx

```diff
@@ -0,0 +1,124 @@
+---
+title: "Query Batching"
+description: "Learn how to use GraphQL query batching in the router."
+icon: layer-group
+sidebarTitle: Query Batching
+---
+
+## Overview
+
+Query batching allows you to send multiple GraphQL operations in a single HTTP request. The router supports processing batched GraphQL requests where multiple operations are sent in a single JSON array. Each operation in the batch is processed concurrently, with configurable limits on the number of operations and concurrency.
+
+<Info>
+ Hereafter, when we refer to a `batch` or `a batch`, we mean the entire batch request. An `operation` or `batch operation` refers to a single record in the batch array that will be processed independently.
+</Info>
+
+<Warning>
+ We do not recommend batching and suggest alternate approaches. Please see the [section below](/router/query-batching#why-we-do-not-recommend-batching) for more information.
+</Warning>
+
+## Configuration
+
+Query batching can be configured in your router configuration file:
+
+```yaml config.yaml
+batching:
+  enabled: true # default: false
+  max_concurrency: 10 # default: 10
```

Comment:
Are you enforcing a value >= 1 in the config?
It should also be documented as such

Reply:
Yes, will add it to the documentation.